IntelliTectAnalyzer/IntelliTectAnalyzer/Analyzers/UnusedLocalVariable.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace IntelliTectAnalyzer.Analyzers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class UnusedLocalVariable : DiagnosticAnalyzer
+    {
+        public const string DiagnosticId = "INTL0303";
+        private const string _Title = "Local variable unused";
+        private const string _MessageFormat = "Local variables should be used";
+        private const string _Description = "All local variables should be used accessed";
```

Comment:
Typo?  Should it be "All local variables should be used" or "All local variables should be accessed"?